index.html

```diff
@@ -342,7 +332,7 @@ <h1>About <br>
           </div>
         </section>
         </main>
-        <section class="contact-me">
+        <section class="contact-me" id="contact">
```

Comment:
- Please make the background of the contact me section takes full width of the webpage as in the Figma design

##### Figma design 
![Screenshot from 2022-06-03 18-37-13](https://user-images.githubusercontent.com/58771507/171917522-9984bd57-ba74-4f23-8be7-f527001d8134.png)


##### Your  design
![Screenshot from 2022-06-03 18-37-32](https://user-images.githubusercontent.com/58771507/171917622-78848043-9858-4f6e-940a-d1b716d42a72.png)